### DISCUSSION TITLE
Globe night and day raster tiles

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-gl-js#4061] [Ideas]
I wonder if it would be possible to show different tiles on a raster globe for night and day areas.

Usually satellite images are taken by day, but there are also these beautiful night-time images which basically show light emitted by human settlements.

Do you think it is possible to blend the two together at the day/night boundary?

**Comment by HarelM** [comment]
In theory, I think you might be able to do that with addProtocol by fetching tiles from different sources and somehow merge tiles on the boundary.
Another option is obviously do the same in the server side, but I'm not aware of a tile server that supports this.

**Comment by louwers** [comment]
Did you ever find out how to do this @wipfli?

**Reply by wipfli** [reply]
I did not
